doc/conf.py

```diff
@@ -19,7 +19,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'SSH-MITM'
-author = 'Manfred Kaiser'
+author = 'Manfred Kaiser, Simon Böhm'

```

Comment:
This will be changed to "SSH-MITM Dev-Team" in the next release

Reply:
We could add a author page, where we can add our github profiles and social links